charts/linkerd2/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: "v1"
-appVersion: edge-19.8.1
+appVersion: edge-19.8.3
```

Comment:
Re: this `appVersion`, and `LinkerdVersion` defined in `values.yaml`, what's the process around changing these? i'm reluctant to set them to versions that have not yet been released, as we won't necessarily know what it's called (`edge-19.8.3`, `edge-19.9.1`, `stable-2.5`, etc).

What about keeping this value as the latest released version, and then rev-ing it when we do release notes / tagging?

Reply:
I would recommend producing the helm chart as a build artifact instead of linking folks to the repo itself. That would allow for updating the app version dynamically and providing what it is based off the tag. Feels like that's part of the work to get to a stable chart as well.